sdk/state/state.go

```diff
@@ -82,9 +81,13 @@ func (c *Channel) SetIterationNumber(i int64) {
 	c.iterationNumber = i
 }
 
-// TODO: Remove
+// Amount returns the amount owing from the initiator to the responder, if positive, or
+// the amount owing from the responder to the initiator, if negative.
 func (c *Channel) Amount() Amount {
```

Comment:
I don't remember why we named this Amount, but we should probably name it Balance.
```suggestion
func (c *Channel) Balance() Amount {
```

Reply:
https://github.com/stellar/experimental-payment-channels/pull/68/commits/acb91cd3780757ea147ce4165662c621e2db3a89